.gitignore

```diff
@@ -9,6 +9,7 @@ yarn-error.log*
 /netbox/netbox/configuration.py
 /netbox/netbox/ldap_config.py
 /netbox/local/*
+/netbox/media/cache
```

Comment:
I was surprised that `/netbox/media` wasn't already in this list. Then I noticed that sub-folders of `/netbox/media` have their own `.gitignore` files.

Any particular reason we didn't use that approach here?

Reply:
IMO it's cleaner and less confusing to keep everything in the root `.gitignore`. I don't think there's any particular reason the other media paths have their own files.

Reply:
I've collapsed them all into the root `.gitignore` per our conversation.

Reply:
AFAIK there's no need to preserve the directories in the repo: Django will create them automatically as needed. Does that impose a challenge for the Docker image? Apologies for the surprise change.

Reply:
It's an easy fix for the image. I just create the media directory in the build process.